src/test/java/org/jabref/logic/importer/fetcher/CiteSeerTest.java

```diff
@@ -8,26 +8,48 @@
 import org.jabref.model.entry.types.StandardEntryType;
 import org.jabref.testutils.category.FetcherTest;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-@Disabled
 @FetcherTest
 class CiteSeerTest {
 
     private CiteSeer fetcher = new CiteSeer();
 
+//    @Test
```

Comment:
I was refering to this commented out code

Reply:
Yes, these are obsolete tests that I had intended to update.